src/app/features/loginReducer.ts

```diff
@@ -1,31 +1,32 @@
-import { Result, TargetElements, requestFromTargetElements } from './login'
-import { useReducer } from 'preact/hooks'
+import { Status, TargetElements, requestFromTargetElements } from './login'
+import { useReducer } from 'react'
```

Comment:
```suggestion
import { useReducer } from 'preact/hooks'
```

여기가 바뀐 이유가 혹시 타입 에러 때문이라면 현재 testing-library를 사용 중이니 되돌려도 좋을 것 같아요.

Reply:
단순한 오타입니다.